onboarding.md

```diff
@@ -245,6 +247,22 @@ needs to be pointed out separately during the onboarding.
   * [`core-validate-commit`][] automates the validation of commit messages.
     This will be run during `git node land --final` of the [`git-node`][]
     command.
+  * Normally you can just use the `commit-queue` label to have the
+    commit queued for landing by the Node.js GitHub bot. But as exercise it is
+    also useful to learn how to land commits manually in case the bot or the CI
+    is broken.
+* If you are landing the commit manually, to make it appear as "Merged" on GitHub,
+  after you prepare the landed commit on the local `main` branch, run this:
+
+  ```bash
+  git checkout your-pr-branch
+  git reset --hard main  # Synchronize your PR branch with the local main branch you just prepared.
+  git push --force-with-lease your-fork-remote your-pr-branch # Update the PR branch in your fork.
+  git checkout main
+  git push upstream main  # Push the landed commit to the upstream main branch.
+  ```
+  GitHub will automatically detect that the PR branch is now identical to the
+  `main` branch and will mark the PR as "Merged".
```

Comment:
Do we want to mention `./tools/actions/merge.sh`?

Reply:
hmm, shouldn't that be implemented in NCU so that there's no need to install gh separately?